test/endpoints/baz/baz_metrics_test.go

```diff
@@ -146,7 +147,77 @@ func TestCallMetrics(t *testing.T) {
 		assert.Equal(t, expectedEndpoitTags[tag.GetTagName()], tag.GetTagValue())
 	}
 
-	tchannelOutboundSuccessMetric := metrics[3]
+	expectedJaegerSpanMetricName := "test-gateway.production.all-workers.jaeger.spans"
```

Comment:
https://github.com/uber/jaeger-client-go/blob/master/config/config.go#L75

^ can set the host port which defaults to `"localhost:6831"` which should just put the host port in the `config/production.json` and overwrite it in the TestGateway / BenchGateway.